### PR TITLE
Ben: OneZero task

### PR DESCRIPTION
**Pull request by bendan7**

### CODE REVIEW

File: lib/interface.tsx [hunk 1]
```diff
@@ -57,6 +49,25 @@ export type Common = {
   footerContactUsTitle: string
   footerScheduleButton: string
   footerSupportUsButton: string
+  contactUsFormErrorEmailInvalid: string
```
Comment (omrirz) [inline lib/interface.tsx:52]: very nice!

File: components/footer.tsx [hunk 2]
```diff
@@ -1,11 +1,50 @@
...
+          <h1 className="text-xl font-bold">{data.footerMenuTitle}</h1>
+          {menuLinks.map((menuLink) => (
+            <Link key={menuLink.text} href={menuLink.relativeLink} passHref>
+              <h1 className={`cursor-pointer text-l `}>{menuLink.text}</h1>
+            </Link>
+          ))}
+        </div>
+        <div className="flex flex-col  justify-self-center">
+          <h1 className="">
+            {data.contactUsLinks.map((data) => (
```
Comment (omrirz) [inline components/footer.tsx]: contactUsLink would be a better name than "data"

**Review by omrirz — CHANGES_REQUESTED** [review]
1. Links are not clickable
Screen Shot 2022-04-25 at 21 57 48
Screen Shot 2022-04-25 at 21 58 12

2. Contact us english
Screen Shot 2022-04-25 at 21 57 34

3. Submit button on mobile not centered
Screen Shot 2022-04-25 at 21 58 39

4. button shape and padding
Screen Shot 2022-04-25 at 21 57 42

5. When clicking "Contact Us" link in header it scrolls and half of the form is out of screen.
Screen Shot 2022-04-25 at 22 24 55

6. use semantic HTML!!!
You got a lot of `<h1>` this is not the way. The page should have one `<h1>`. each section title should have `<h2>`. Links should be `<a>` and not `<h1>`. use the correct semantic element for each task.

7. If you went and put in the effort into mobile navbar (which is nice). go ahead and close it when a language is chosen. also I would make the body non scrollable when the menu is open. Do you think the sliding menu can slide nicely with an animation instead of just popping up?

**Comment by bendan7** [comment]
opsss most of the issues were in the last commit that I forgot to push.

**Comment by bendan7** [comment]
Hi Omri, first of all, I want to thank you for the code review, it's really not obvious these days.
Regarding the notes, I pushed the commit that I forgat which contains fixes for most of the issues and I added 1 more commit for the Btn issue in the ContactUsForm. regarding the RIGHT markups and the animation issues, I hope I will find the time in the next 2-3 to finish the task.

**Comment by omrirz** [comment]
@bendan7 the animation is of lesser importance. The semantic HTML is CRITICAL! We build 100% accessible products. Nothing less.

**Comment by bendan7** [comment]
I didn't know that is so important, I will read about that. thanks

**Comment by bendan7** [comment]
@omrirz  All comments have been fixed

File: components/footer.tsx [hunk 3]
```diff
@@ -31,19 +31,19 @@ export default function Footer({ data }: CommonType) {
           />
         </div>
         <div className="flex flex-col  justify-self-center">
-          <h1 className="text-xl font-bold">{data.footerMenuTitle}</h1>
+          <h2 className="text-xl font-bold">{data.footerMenuTitle}</h2>
           {menuLinks.map((menuLink) => (
             <Link key={menuLink.text} href={menuLink.relativeLink} passHref>
-              <h1 className={`cursor-pointer text-l `}>{menuLink.text}</h1>
+              <h2 className={`cursor-pointer text-l `}>{menuLink.text}</h2>
```
Comment (omrirz) [inline components/footer.tsx]: This looks like a `<ul>` with `<li>` and Links. those are not headers.

File: components/Navigation.tsx [hunk 4]
```diff
@@ -0,0 +1,142 @@
...
+  appLinks: Array<AppLink>
+  className?: string
+  onClick?: Function
+}) => {
+  return (
+    <>
+      {appLinks.map(({ text, relativeLink }) => (
+        <Link key={text} href={relativeLink || '#top-header'} passHref>
+          <div onClick={(e) => onClick && onClick()} className="hover:bg-azure">
+            <h2 className={`cursor-pointer text-xl ${className}`}>{text}</h2>
```
Comment (omrirz) [inline components/Navigation.tsx]: those are `<li>` and not `<h2>`

File: components/footer.tsx [hunk 5]
```diff
@@ -1,11 +1,50 @@
...
+        <div className="flex flex-col  justify-self-center">
+          <h2 className="text-xl font-bold">{data.footerMenuTitle}</h2>
+          {menuLinks.map((menuLink) => (
+            <Link key={menuLink.text} href={menuLink.relativeLink} passHref>
+              <h2 className={`cursor-pointer text-l `}>{menuLink.text}</h2>
+            </Link>
+          ))}
+        </div>
+        <div className="flex flex-col  justify-self-center">
+          <h2 className="">
```
Comment (omrirz) [inline components/footer.tsx]: all the links together are an h2?

File: components/home-components/what-we-do.tsx [hunk 6]
```diff
@@ -1,7 +1,60 @@
...
-  return <WrapperLarge>What We Do </WrapperLarge>
+  return (
+    <WrapperLarge id="about-us" className="mx-auto max-w-screen-xl">
+      <div className="text-center text-red text-2xl">
+        {data.homepage.whatWeDoHeading}
+      </div>
+      <div className="text-center font-bold text-6xl mb-14">
+        {data.homepage.whatWeDoTitle}
+      </div>
+      <div className="grid gap-4 lg:grid-cols-4 md:grid-cols-2 sm:grid-cols-2 sm:gap-0">
```
Comment (omrirz) [inline components/home-components/what-we-do.tsx]: use a `<ul>` and not just div everywhere.

**Review by omrirz — CHANGES_REQUESTED** [review]

File: components/footer.tsx (hunk 5 above)
Comment (bendan7) [reply components/footer.tsx]: My bad, fixed

**Comment by bendan7** [comment]
@omrirz All comments have been fixed 

+ I Added on-hover animation to the Button component.

**Pull request closed without merging**